### ISSUE TITLE
get_connection crashes when sniff_hosts enabled in version 1.3.0

### ISSUE THREAD

**Issue by smoy** [elasticsearch/elasticsearch-py#180]
```
>>> import elasticsearch
>>> es = elasticsearch.Elasticsearch([{'host': 'my-es-host', 'port': 9300}], sniff_on_start=True, sniff_on_connection_fail=True, sniffer_timeout=5)
>>> es.transport.get_connection()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/nail/home/smoy/yelp-git/dowser/.tox/py26/lib/python2.6/site-packages/elasticsearch/transport.py", line 182, in get_connection
    self.sniff_hosts()
  File "/nail/home/smoy/yelp-git/dowser/.tox/py26/lib/python2.6/site-packages/elasticsearch/transport.py", line 198, in sniff_hosts
    for c in self.connection_pool.connections + self.seed_connections:
TypeError: can only concatenate list (not "tuple") to list
>>> es.transport.seed_connections
(<Urllib3HttpConnection: http://my-es-host:9300>,
```

**Comment by HonzaKral** [comment]
Thanks for the report! I cannot seem to replicate it but I think I have discovered the issue and fixed it, could you please try again?

**Comment by smoy** [comment]
I think there is still a bug: open pull request 181

https://github.com/elasticsearch/elasticsearch-py/pull/181

**Comment by teuneboon** [comment]
Having the same problem in 1.4.2, in case that helps you with debugging. Although it seems like it's fixed already in pull request #181
